applications/DeepSpeed-Chat/setup.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+# setup.py: install script for deepspeed_chat
+"""
+to install deepspeed_chat and its dependencies for development work,
+run this cmd from the root directory:
+    pip install -e .
+"""
+import setuptools
+
+setuptools.setup(
+    name="deepspeed_chat",
+    version="0.1",
+    url=
+    "https://github.com/microsoft/DeepSpeedExamples/tree/master/applications/DeepSpeed-Chat",
+    include_package_data=True,
+    packages=setuptools.find_packages(include=['deepspeed_chat']),
```

Comment:
Let's name the package deepspeed-chat (There is a package named deepspeed-mii) so let's follow the convention of hyphen instead of underscore.

Reply:
@jeffra and I discussed this a bit. Let me write what we decided here. PyPI name will be deepspeed-chat and the python package will be dschat since python imports won't work with hyphens